site2/docs/reference-configuration.md

```diff
@@ -657,6 +657,7 @@ The value of 0 disables message-byte dispatch-throttling.|0|
 |defaultRetentionSizeInMB|    |0|
 |keepAliveIntervalSeconds|    |30|
 haProxyProtocolEnabled | Enable or disable the [HAProxy](http://www.haproxy.org/) protocol. |false|
+bookieId | If you want to custom bookie ID or use a dynamic network address for the bookie, you can set this option. <br><br>Bookie advertises itself using `bookieId` rather than `BookieSocketAddress` (hostname:port or IP:port).<br><br> `bookieId` is a non empty string that can contain ASCII digits and letters ([a-zA-Z9-0]), colons, dashes, and dots. <br><br>For more information about `bookieId`, see [here](http://bookkeeper.apache.org/bps/BP-41-bookieid/).|/|
```

Comment:
same as above

Reply:
`useHostNameAsBookieID` is not in the `standalone.conf` file, so I do not move this line.